src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
I think we want to use the new 6.4 routers. Check out this one -> https://reactrouter.com/en/main/routers/create-browser-router#createbrowserrouter